terasoluna-gfw-libraries-parent/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>terasoluna-gfw-libraries-parent</artifactId>

```

Comment:
@btyoshikawaa this comment correspond to f203fe4d1b30478357c925eba5000421bf73c27e.
I restructured directory as below.
```
terasoluna-gfw ( module root )
  - terasoluna-gfw-parent ( parent )
  - terasoluna-gfw-common-libraries ( module root and parent )
    - terasoluna-gfw-common ...and others...
  - terasoluna-gfw-dependencies ( module root and parent )
    - terasoluna-gfw-common-dependencies ...and others...
```